src/containers/MultidisciplinarySubject/components/MultidisciplinarySubjectArticle.tsx

```diff
@@ -207,13 +205,24 @@ const MultidisciplinarySubjectArticle = ({ topic, subject, resourceTypes, skipTo
 
 export const multidisciplinarySubjectArticleFragments = {
   topic: gql`
-    fragment MultidisciplinarySubjectArticle_Topic on Topic {
-      path
+    fragment MultidisciplinarySubjectArticle_Topic on Node {
```

Comment:
Kanskje denne også burde hete `parent`, for å være konsistent med dataen den bygger på?